src/sparseml/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -112,16 +119,31 @@ def save_pretrained_wrapper(
             # state_dict gets passed in as a kwarg for FSDP models
             state_dict = kwargs.get("state_dict", None)
 
-            if qat_active(model):
+            if qat_active(model) or is_model_quantized(model):
                 _LOGGER.info(
                     "Compression for quantized models is not yet supported. Save will "
                     "be run without compression and no sparsity statistics will be "
                     "calculated."
                 )
-                return original_save_pretrained.__get__(model, model_class)(
+
+                original_save_pretrained.__get__(model, model_class)(
                     save_directory, **kwargs
                 )
 
+                if is_model_quantized(model):
+                    quant_config = QuantizationConfig.from_pretrained(model)
```

Comment:
let's unify the initialization of configs, I'd be down for QuantizationConfig.from_pretrained(model) and SparsificationConfig.from_pretrained(model) (instead of currently using a helper func `infer_sparsification_config_from_model`